tests/ssh-priv-key/key

```diff
@@ -0,0 +1,51 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
these keys are generated for the purpose of this PR?

Reply:
I copied them from [mender-test-containers](https://github.com/mendersoftware/mender-test-containers/tree/master/docker/ssh-keys).